grails-app/views/species/show.gsp

```diff
@@ -1329,9 +1331,9 @@
                 scholarUrl:         "${createLink(controller: 'externalSite', action: 'scholar', params: [s: tc?.taxonConcept?.nameString ?: ''])}",
                 soundUrl:           "${createLink(controller: 'species', action: 'soundSearch', params: [s: tc?.taxonConcept?.nameString ?: ''])}",  // FixMe: do somthing so that it starts working
                 eolLanguage:        "${grailsApplication.config.eol.lang}",
-                defaultDecimalLatitude: ${grailsApplication.config.defaultDecimalLatitude},
-                defaultDecimalLongitude: ${grailsApplication.config.defaultDecimalLongitude},
-                defaultZoomLevel: ${grailsApplication.config.defaultZoomLevel},
+                defaultDecimalLatitude: "${grailsApplication.config.defaultDecimalLatitude}",
+                defaultDecimalLongitude: "${grailsApplication.config.defaultDecimalLongitude}",
+                defaultZoomLevel: "${grailsApplication.config.defaultZoomLevel}",
```

Comment:
This makes them strings. JS will probably auto-convert to numbers when they are used, but imho should leave them with correct types.